### ISSUE TITLE
finding the nodeId of a node by Browse name

### ISSUE THREAD

**Issue by UmerIftikhar** [node-opcua/node-opcua#218]
Does this functionality has been added?
As it says here:
https://github.com/node-opcua/node-opcua/blob/master/documentation/creating_a_client.md

**Comment by erossignon** [comment]
Yes the TranslateBrowsePathsToNodeIds service has been implemented.
Have a [look at the unit test to discover how it can be used](
https://github.com/node-opcua/node-opcua/blob/4f2538d1580d35f463a7b8d66abb5f281ae97828/test/end_to_end/u_test_e2e_translateBrowsePath.js#L41)

**Comment by prenaux** [comment]
It doesn't seem to handle spaces correctly.

When I try to resolve: "/Objects/2:MatrikonOPC Simulation Server (DA)" it throws:
```
Error: Malformed relative path  Simulation Server (DA)Error: Malformed relative path  Simulation Server (DA)
 at makeRelativePath (d:\Work\www-ts-opc\node_modules\node-opcua\lib\address_space\make_relative_path.js:127:19)
    at makeBrowsePath (d:\Work\www-ts-opc\node_modules\node-opcua\lib\address_space\make_browse_path.js:19:23)
``

Resolving "/Objects/Server", etc... works as expected.

**Comment by prenaux** [comment]
Why was this closed? was it fixed or moved to another issue?

**Comment by erossignon** [comment]
Yep! this has been fixed and tested in commit 801c87175436aca5ebaeaa2c9393fc11d0cca13f . 
I guess Github automatically closed the issue for us because the  the comment in the commit contained "fixes #218 ..." .

**Comment by prenaux** [comment]
I can confirm it works, thanks a lot :)
